edge/pkg/common/dbm/db.go

```diff
@@ -72,9 +73,20 @@ func InitDBManager() {
 	DBAccess.Using(dbName)
 }
 
+// CleanDBFile removes db file if it is neccessary
+func CleanDBFile() {
```

Comment:
This function is only used to remove `defaultDataSource` file?

Why not pass a file argument in?

Reply:
I will Thanks